src/main/scala/com/typesafe/sbt/packager/docker/Keys.scala

```diff
@@ -31,7 +31,7 @@ trait DockerKeys {
   val dockerExecCommand = SettingKey[Seq[String]]("dockerExecCommand", "The shell command used to exec Docker")
   val dockerBuildOptions = SettingKey[Seq[String]]("dockerBuildOptions", "Options used for the Docker build")
   val dockerBuildCommand = SettingKey[Seq[String]]("dockerBuildCommand", "Command for building the Docker image")
-  val dockerLabels = SettingKey[Map[String, String]]("dockerLabels", "Labels applied to the Docker image")
+  val dockerLabels = SettingKey[Map[String, Any]]("dockerLabels", "Labels applied to the Docker image")
```

Comment:
`Any` is not a good type here. If the number/string separation is really necessary I would rather map this into a separate ADT. We then need to introduce implicit conversions to not break existing builds.